dns/zone-configs/k8s.io.yaml

```diff
@@ -0,0 +1,216 @@
+  value: redirect.k8s.io.
+dl:
+  type: CNAME
+  value: redirect.k8s.io.
+rel:
+  type: CNAME
+  value: redirect.k8s.io.
+releases:
+  type: CNAME
+  value: redirect.k8s.io.
+
+# Vanity CNAMEs.
+blog:
+  type: CNAME
+  value: redirect.k8s.io.
+changelog:
+  type: CNAME
+  value: redirect.k8s.io.
+# TODO: Where does this live and who owns it?
+cherrypick:
+  type: A
+  value: 35.184.147.212
+ci-test:
+  type: CNAME
+  value: redirect.k8s.io.
+code:
+  type: CNAME
+  value: redirect.k8s.io.
+contributor:
+  type: CNAME
+  value: kubernetes-contributor.netlify.com.
+cs:
+  type: A
+  value: 147.75.97.58
+devstats:
+  type: CNAME
+  value: k8s.devstats.cncf.io.
+# TODO: jorge and paris and which sig?
+discuss:
+  type: CNAME
+  value: kubernetes.hosted-by-discourse.com.
+examples:
+  type: CNAME
+  value: redirect.k8s.io.
+feature:
+  type: CNAME
+  value: redirect.k8s.io.
+features:
+  type: CNAME
+  value: redirect.k8s.io.
+# Web frontend for unauthenticated GCS access.  Running in GKE (@thockin).
+gcsweb:
+  type: A
+  value: 104.197.177.166
+get:
+  type: CNAME
+  value: redirect.k8s.io.
+git:
+  type: CNAME
+  value: redirect.k8s.io.
+go:
+  type: CNAME
+  value: redirect.k8s.io.
+gubernator:
+  type: CNAME
+  value: redirect.k8s.io.
+issue:
+  type: CNAME
+  value: redirect.k8s.io.
+issues:
+  type: CNAME
+  value: redirect.k8s.io.
+# Running in GKE (@dchen1107).
+node-perf-dash:
+  type: A
+  value: 130.211.155.47
+# Who owns this and where is it running?
+perf-dash:
+  type: A
+  value: 35.188.102.189
+pr:
+  type: CNAME
+  value: redirect.k8s.io.
+pr-test:
+  type: CNAME
+  value: redirect.k8s.io.
+# Who owns this and where is it running?
+prow:
+  type: A
+  value: 35.186.196.185
+prs:
+  type: CNAME
+  value: redirect.k8s.io.
+sigs:
+  type: CNAME
+  value: redirect.k8s.io.
+# Who owns this and where did we get that IP?
+slack:
+  type: A
+  value: 104.197.79.9
+# Voluntary cluster data collector. Runs in GKE (@thockin).
+spartakus:
+  type: A
+  value: 107.178.240.20
+# Prow (@ixdy).
+submit-queue:
+  type: CNAME
+  value: redirect.k8s.io.
+# Who owns this and do we need it?
+test-aws:
+  type: NS
+  values:
+  - ns-1345.awsdns-40.org.
+  - ns-1710.awsdns-21.co.uk.
+  - ns-491.awsdns-61.com.
+  - ns-969.awsdns-57.net.
+# Who owns this and do we need it?
+test-cncf-aws:
+  type: NS
+  values:
+  - ns-1458.awsdns-54.org.
+  - ns-1825.awsdns-36.co.uk.
+  - ns-265.awsdns-33.com.
+  - ns-687.awsdns-21.net.
+# Who owns this and do we need it?
+test-e2e:
+  type: NS
+  values:
+  - ns-cloud-a1.googledomains.com.
+  - ns-cloud-a2.googledomains.com.
+  - ns-cloud-a3.googledomains.com.
+  - ns-cloud-a4.googledomains.com.
+# Who owns this and do we need it?
+test-gcp:
+  type: NS
+  values:
+  - ns-cloud-e1.googledomains.com.
+  - ns-cloud-e2.googledomains.com.
+  - ns-cloud-e3.googledomains.com.
+  - ns-cloud-e4.googledomains.com.
+testgrid:
+  type: CNAME
+  value: redirect.k8s.io.
+# Project metrics.  Running in GKE (@apelisse, @jgrafton)
```

Comment:
At this point I would swap in myself or @cjwagner for this

Reply:
ACK